### DISCUSSION TITLE
Add #shorts type

### DISCUSSION THREAD

**Discussion by pepsiamir** [DrKain/scrape-youtube#68] [Ideas]
This package is awesome. Youtube shows #shorts in the same results page in a seperate section.
That would be awesome, if we could scrape shorts too.

Thanks

**Comment by DrKain** [comment]
This may have changed but when shorts were introduced you could simply add `"#shorts"` to the search query. https://github.com/DrKain/scrape-youtube/issues/54#issuecomment-1019286144  
I'm away from home so I'm not able to test to make sure this is still valid, but I'll make a note to take another look at it when I can

**Reply by pepsiamir** [reply]
Yes, That's right some people put #shorts hashtag on title or description to let YT know that this video is short. Your tool can find some of those. but YT has a dedicated carousel grid for shorts on the result page. If scraper-youtube can scrape that div, it would be great.
again Thank you for your awesome tool

**Reply by DrKain** [reply]
Sure thing, I'll see what I can do when I get the spare time

**Reply by DevJuliox** [reply]
Quisiera ayudar con esto, necesito filtrar por shorts y videos
